hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/CopyCommands.java

```diff
@@ -329,6 +335,22 @@ private void setNumberThreads(String numberThreadsString) {
       }
     }
 
+    private void setThreadPoolQueueSize(String numThreadPoolQueueSize) {
+      if (numThreadPoolQueueSize != null) {
+        int parsedValue = Integer.parseInt(numThreadPoolQueueSize);
+        if (parsedValue < 1) {
+          threadPoolQueueSize = 1024;
```

Comment:
Better to add a warn log line here, to denote the thread pool queue size was misconfigured and we are now using the default.

Reply:
Thanks @ayushtkn  for the comment.
I will make some changes later.